Tools/setup/ubuntu.sh

```diff
@@ -118,38 +126,68 @@ if [[ $INSTALL_NUTTX == "true" ]]; then
 		build-essential \
 		flex \
 		g++-multilib \
-		gcc-arm-none-eabi \
```

Comment:
You're reverting multiple improvements of https://github.com/PX4/PX4-Autopilot/pull/23869